package ch04.ex02.case01;

public class ArrayException {
	public static void main(String[] args) {
		
		int[] arr = new int[10];
		int result = arr[10];
		
		System.out.println("끝");
	}
}
